example-dapp/scripts/simulateTx.ts

```diff
@@ -14,7 +14,7 @@ async function main() {
 
   const dapp = await deployKrnlDapp(tokenAuth, signer);
 
-  const faasRequests: string[] = ["KYT", "KYC"]
+  const faasRequests: string[] = ["KYT", "KYC", "EL_KYT"]
```

Comment:
add comment pls for EL_KYT that the eigen stuff needs to be up and running + provide the link for the repo